src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, h, Prop, Watch, State } from "@stencil/core";
+
+@Component({
+  tag: "radio-group",
+  styleUrl: "radio-group.css",
+  shadow: true,
+})
+export class RadioGroup {
+  @Prop({ reflect: true }) list: Array<string>;
+  @State() selectedIndex: number;
+  @State() selectedItem: string;
+
+  handleChange = (e: any) => {
+    this.selectedIndex = parseInt(e.target.value);
+    this.selectedItem = e.target.name;
+  };
+
+  render() {
+    return (
+      <Host>
```

Comment:
@anveshmekala can you explain why you added the `<host>` tag here? If you're not modifying the host, i don't think it is necessary.

Reply:
`<Host>` tag is added as a fragment here . I chose it to keep the code consistent rather than wrapping up the JSX part inside a `<div>` tag or any other html tag as parent.